base_user_role/data/ir_cron.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 ABF OSIELL <http://osiell.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo noupdate="1">
+
+<record model="ir.cron" id="cron_update_users">
+    <field name='name'>Update user roles</field>
+    <field name='interval_number'>3</field>
+    <field name='interval_type'>hours</field>
+    <field name="numbercall">-1</field>
+    <field name="active">True</field>
+    <field name="doall" eval="False"/>
+    <field name="model_id" ref="base_user_role.model_res_users_role"/>
+    <field name="state">code</field>
+    <field name="code">model.cron_update_users()</field>
+    <field name="args">()</field>
```

Comment:
Runbot is failing due to this:

> 2017-11-23 17:23:24,700 151 WARNING openerp_test odoo.models: ir.cron.create() includes unknown fields: args

https://runbot2.odoo-community.org/runbot/static/build/3311019-1-0b09c6/logs/job_20_test_all.txt

Reply:
You're right, i fixed. Thank you very much (y)

Reply:
No prob! Admittedly I haven't reviewed beyond this, I just figured I'd help out where I could.

Looks like we're good for merge though, so I'll hit the button

Reply:
So if there is nothing else, can you approve the PR?

Reply:
My bad, I forgot to hit the button 😑